src/core/config/trel.h

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright (c) 2019, The OpenThread Authors.
+ *  All rights reserved.
+ *
+ *  Redistribution and use in source and binary forms, with or without
+ *  modification, are permitted provided that the following conditions are met:
+ *  1. Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *  2. Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in the
+ *     documentation and/or other materials provided with the distribution.
+ *  3. Neither the name of the copyright holder nor the
+ *     names of its contributors may be used to endorse or promote products
+ *     derived from this software without specific prior written permission.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ *  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ *  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ *  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ *  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ *  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ *  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ *  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ *  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ *  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *  POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/**
+ * @file
+ *   This file includes compile-time configurations for TREL.
+ *
+ */
+
+#ifndef CONFIG_TREL_H_
+#define CONFIG_TREL_H_
+
+/**
+ * @def OPENTHREAD_CONFIG_TREL_PEER_TABLE_SIZE
+ *
+ * Specifies the capacity of TREL peer table.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_TREL_PEER_TABLE_SIZE
+#define OPENTHREAD_CONFIG_TREL_PEER_TABLE_SIZE (50)
+#endif
+
+/**
+ * @def OPENTHREAD_CONFIG_TREL_TX_PACKET_POOL_SIZE
+ *
+ * Specifies the capacity of TREL packet pool for transmission.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_TREL_TX_PACKET_POOL_SIZE
+#define OPENTHREAD_CONFIG_TREL_TX_PACKET_POOL_SIZE (10)
+#endif
+
+/**
+ * @}
+ *
+ */
```

Comment:
```suggestion
```

Reply:
I noticed a pattern that each config file has this block in the end. Should we follow this?

Reply:
`@}` is a closing bracket matching with `@{`. When there's no `@{` you shouldn't add it.

Reply:
I added the `@brief` block with ` @{` above.